language-plutus-core/src/Language/PlutusCore/CBOR.hs

```diff
@@ -76,8 +76,6 @@ instance Serialise BuiltinName where
                 SHA3                 -> 17
                 VerifySignature      -> 18
                 EqByteString         -> 19
-                TxHash               -> 20
-                BlockNum             -> 21
                 SizeOfInteger        -> 22
                 QuotientInteger      -> 23
                 ModInteger           -> 24
```

Comment:
Should I update the numbering here or do we already care about backwards compatibility?

Reply:
I think we care about backwards compatibility at precisely the point where this stuff goes on the real chain, so not now.

Reply:
So yes I think the numbering should change.